velox/exec/fuzzer/WindowFuzzer.cpp

```diff
@@ -242,16 +254,23 @@ void WindowFuzzer::go() {
           generateSortingKeysAndOrders("s", argNames, argTypes);
     }
     const auto partitionKeys = generateSortingKeys("p", argNames, argTypes);
-    const auto [frameClause, isRowsFrame] = generateFrameClause();
-    const auto input =
-        generateInputDataWithRowNumber(argNames, argTypes, signature);
+
+    // Randomly select if ROWS or RANGE frame
+    auto typeOfFrame = vectorFuzzer_.coinToss(0.5) ? 1 : 0;
```

Comment:
I suggest to define two new constants kRowTypeFrame and kRangeTypeFrame and set them here.
Later you don't have to compare with 0 or 1 but can use the constants again. You'd want names for easier readability of the code.

Reply:
Will define on top in WindowFuzzer.h:

```
const int frameTypeRange = 0;
const int frameTypeRow = 1;
```

Reply:
Using the following below since it's already defined
```
core::WindowNode::WindowType::kRows
core::WindowNode::WindowType::kRange
```